pallets/parachain-staking/src/weights.rs

```diff
@@ -184,6 +184,8 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 			.saturating_add((73_000 as Weight).saturating_mul(x as Weight))
 			.saturating_add(T::DbWeight::get().reads(5 as Weight))
 			.saturating_add(T::DbWeight::get().writes(6 as Weight))
+			// Storage: Session NextKeys (r:1 w:0) -- ValidatorRegistration
+			.saturating_add(T::DbWeight::get().reads(1 as Weight))
```

Comment:
Opted to add separate line here rather than including the weight in L185 to avoid this change getting clobbered by future upstream changes that increase the existing reads from `5->6`.

Is there anything else that I should be doing wrt benchmarking/weight?

Reply:
ahh this file is normally auto-generated by running the benchmarks. is the benchmark returning this, or did you just add the line to this?

Reply:
Opting, to leave this manual change in for now. I'm still not sure how best to manage generating these weights in an external repo. Hoping that this one line change is the least overhead for now (assuming that it doesn't get wiped out by merge requests).